package.json

```diff
@@ -17,9 +17,13 @@
   },
   "dependencies": {
     "code-push": "1.5.2-beta",
-    "semver": "^5.1.0"
+    "semver": "^5.1.0",
+    "rnpm": "1.4.1"
   },
-  "devDependencies": {
-    "react-native": "0.19.0"
+  "peerDependencies": {
+    "react-native": "^0.19.0"
+  },
+  "scripts": { 
+    "postinstall": "cd ../../ && rnpm link react-native-code-push"
```

Comment:
Yup, that was fixing the issue with detecting that a project has already been linked. I have another PR there now that fixes that issue with single line multi project import statements. https://github.com/rnpm/rnpm-plugin-link/pull/31